server/views/applications/withdrawals/new.njk

```diff
@@ -25,6 +25,12 @@
 
         {{ showErrorSummary(errorSummary) }}
 
+          <ul class="govuk-list govuk-list--bullet">
```

Comment:
We should check that the notes exist before creating the `ul` otherwise we will be adding an empty list to the markup